install.sh

```
@@ -0,0 +1,287 @@
+        fish)
+            echo "$HOME/.config/fish/config.fish"
+            ;;
+        *)
+            echo "$HOME/.profile"
+            ;;
+    esac
+}
+
+# Function to add directory to PATH
+add_to_path() {
+    local profile_file=$(detect_shell_profile)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)